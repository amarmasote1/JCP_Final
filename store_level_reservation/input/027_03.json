[
  {
    "storeID": "0696",
    "itemID": "12345678043",
    "uom": "EACH",
    "supplyData": {
      "onhandsupply":100
    },
    "demandData": {
        "reserveddemand": 0
      },
    "timestamp": "<$currentTime>"
    
  }
]