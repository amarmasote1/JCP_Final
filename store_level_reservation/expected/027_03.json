{"status":200,"message":"Success","body":[{"itemID":"12345678043","uom":"EACH","storeID":"0696","supplyData":{"onhandsupply":100},"demandData":{"reserveddemand":0}}]}